deepmd/descriptor/se_a_mask.py

```diff
@@ -0,0 +1,833 @@
+import math
+import warnings
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+    Tuple,
+)
+
+import numpy as np
+
+from deepmd.common import (
+    get_activation_func,
+    get_precision,
+)
+from deepmd.env import (
+    GLOBAL_NP_FLOAT_PRECISION,
+    GLOBAL_TF_FLOAT_PRECISION,
+    default_tf_session_config,
+    op_module,
+    tf,
+)
+from deepmd.utils.argcheck import (
+    list_to_doc,
+)
+from deepmd.utils.graph import (
+    get_tensor_by_name_from_graph,
+    load_graph_def,
+)
+from deepmd.utils.network import (
+    embedding_net,
+    embedding_net_rand_seed_shift,
+)
+from deepmd.utils.sess import (
+    run_sess,
+)
+from deepmd.utils.tabulate import (
+    DPTabulate,
+)
+from deepmd.utils.type_embed import (
+    embed_atom_type,
+)
+
+from .descriptor import (
+    Descriptor,
+)
+from .se import (
+    DescrptSe,
+)
+
+
+@Descriptor.register("se_a_mask")
+class DescrptSeAMask(Descriptor):
```

Comment:
To make `dp compress` work, you have to make `DescrptSeAMask` inherit from `DescrptSeA`. Is every method in `DescrptSeAMask` different from `DescrptSeA`?

Reply:
`comput_input_stats` is not supported in se_a_mask. 
Thus, `merge_input_stats` and `enable_compression` may not supported either.

Reply:
In the latest commit. DescrptSeAMask is inherit from DescrptSeA.
Methods `get_rcut`, `compute_input_stats`, `build`, `prod_force_virial` are overloaded. 
However, I can't verify if `dp compress` works with `se_a_mask`.